src/main/java/com/bskura/myorganizerapi/controller/ProjectController.java

```diff
@@ -0,0 +1,39 @@
+package com.bskura.myorganizerapi.controller;
+
+import com.bskura.myorganizerapi.entity.Project;
+import com.bskura.myorganizerapi.service.ProjectService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping(path = "projects")
+public class ProjectController {
+    private final ProjectService projectService;
+
+    @Autowired
+    public ProjectController(ProjectService projectService) {
+        this.projectService = projectService;
+    }
+
+    @PostMapping
+    public void createProject(@RequestBody Project project) {
+        projectService.createProject(project);
+    }
+
+    @GetMapping
+    public List<Project> getProjects() {
+        return projectService.getProjects();
+    }
+
+    @PutMapping(path="{id}")
+    public void updateProject(@PathVariable("id") Long id, @RequestParam(required = false) String name, @RequestParam(required = false) String description) {
+        projectService.updateProject(id, name, description);
+    }
```

Comment:
The whole endpoint (included the part in projectService) doesn't look good for, I would replace it with PATCH /endpoints/{id} and I would use body parameters instead of path parameters. It'll be done in other pull request.